package.json

```diff
@@ -1,28 +1,29 @@
-    "example-lifecycle": "pulp build --main Example.Lifecycle.Main --include examples/lifecycle/src --to examples/lifecycle/dist/example.js"
+    "clean": "rimraf output && rimraf .spago",
+    "postinstall": "spago install",
+    "build": "spago build --purs-args '--censor-lib --strict'",
+    "test": "spago --config test/test.dhall test --purs-args '--censor-lib --strict'",
+    "example-all": "./build-examples.sh",
+    "example-ace": "spago --config examples/ace/spago.dhall bundle-app --main Example.Ace.Main --to examples/ace/dist/example.js",
+    "example-basic": "spago --config examples/basic/spago.dhall bundle-app --main Example.Basic.Main --to examples/basic/dist/example.js",
+    "example-components": "spago --config examples/components/spago.dhall bundle-app --main Example.Components.Main --to examples/components/dist/example.js",
+    "example-components-inputs": "spago --config examples/components-inputs/spago.dhall bundle-app --main Example.Components.Inputs.Main --to examples/components-inputs/dist/example.js",
+    "example-components-multitype": "spago --config examples/components-multitype/spago.dhall bundle-app --main Example.Components.Multitype.Main --to examples/components-multitype/dist/example.js",
+    "example-driver-io": "spago --config examples/driver-io/spago.dhall bundle-app --main Example.Driver.IO.Main --to examples/driver-io/dist/example.js",
+    "example-driver-routing": "spago --config examples/driver-routing/spago.dhall bundle-app --main Example.Driver.Routing.Main --to examples/driver-routing/dist/example.js",
+    "example-driver-websockets": "spago --config examples/driver-websockets/spago.dhall bundle-app --main Example.Driver.Websockets.Main --to examples/driver-websockets/dist/example.js",
+    "example-effects-aff-ajax": "spago --config examples/effects-aff-ajax/spago.dhall bundle-app --main Example.Effects.Aff.Ajax.Main --to examples/effects-aff-ajax/dist/example.js",
+    "example-effects-effect-random": "spago --config examples/effects-effect-random/spago.dhall bundle-app --main Example.Effects.Effect.Random.Main --to examples/effects-effect-random/dist/example.js",
+    "example-higher-order-components": "spago --config examples/higher-order-components/spago.dhall bundle-app --main Example.HOC.Main --to examples/higher-order-components/dist/example.js",
+    "example-interpret": "spago --config examples/interpret/spago.dhall bundle-app --main Example.Interpret.Main --to examples/interpret/dist/example.js",
+    "example-keyboard-input": "spago --config examples/keyboard-input/spago.dhall bundle-app --main Example.KeyboardInput.Main --to examples/keyboard-input/dist/example.js",
+    "example-lifecycle": "spago --config examples/lifecycle/spago.dhall bundle-app --main Example.Lifecycle.Main --to examples/lifecycle/dist/example.js"
   },
   "devDependencies": {
-    "pulp": "^13.0.0",
-    "purescript": "^0.13.0",
```

Comment:
I think I'd rather keep PS as an explicit dependency rather than grabbing the latest. 

Although we've just switched over all the core/contrib repos to grab the latest compiler, that was kinda leftover from the days where we weren't free to make an npm release whenever we wanted. I think depending on a specific compiler version is preferable for libraries, rather than always grabbing the latest, since it might well be that there's a window where we're not compatible with the latest compiler. It can especially happen in projects with lots of dependencies, since everything upstream needs to be ready also.

Reply:
Works for me! I've added PureScript back as a dev dependency and reverted that part of the Travis script.